applications/sasquatch/charts/app-metrics/values.yaml

```diff
@@ -26,6 +26,9 @@ globalAppConfig:
   nublado:
     influxTags:
       - "username"
+  sia:
+    influxTags:
+      - "success"
```

Comment:
You can get rid of this now that you split querying into separate success and failure events

Reply:
Actually I need to figure out what tags we want in this case, since as I understand it, adding tags here is required for enabling the sasquatch metrics.

 Since my events only record duration (and error in the case of failed queries) does it make sense to have a duration tag? Or perhaps should I add a username to the event and have that be the influxTag configured here?

Reply:
I didn't think you needed to specify any tags if you don't want any... is it not applying correctly if you don't have any tags speficied?

Reply:
Ahh I see, it's assuming there's an entry for ever app specied in `.Values.apps`. I can fix that...

Reply:
Ok, in the meantime I'm also adding a username to the event here:
https://github.com/lsst-sqre/sia/pull/34

So I may actually just add that as a tag for sia in sasquatch